patches/chrome-browser-extensions-api-webrtc_logging_private-webrtc_logging_private_api.cc.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc b/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc
-index 32845ee2e0e9677d7d2194e31f0bb2718c925bd4..18381a3c61b45879397836f23543d1d642a3ee24 100644
+index 32845ee2e0e9677d7d2194e31f0bb2718c925bd4..2146214dced7a1f54feb75b91717d876b7b7b511 100644
 --- a/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc
 +++ b/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc
 @@ -89,6 +89,7 @@ content::RenderProcessHost* WebrtcLoggingPrivateFunction::RphFromRequest(
      const api::webrtc_logging_private::RequestInfo& request,
      const std::string& security_origin,
      std::string* error) {
-+  return nullptr; // feature disabled in Brave
++  BRAVE_DISABLE_RPH_FROM_REQUEST //feature disabled in Brave
```

Comment:
Will be reverted when this is merged: https://github.com/brave/brave-core/pull/6522